### PR TITLE
[10.0][MIG] sale_delivery_split_date module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: sale_delivery_split_date/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = 'stock.picking'
+
+    min_dt = fields.Date(
+        strint='Scheduled Date (for filter purpose only)',
```
Comment (JayVora-SerpentCS) [inline sale_delivery_split_date/models/stock_picking.py]: string, misspelled.

File: sale_delivery_split_date/views/stock_picking.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Alex Comba - Agile Business Group
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<openerp>
```
Comment (JayVora-SerpentCS) [inline sale_delivery_split_date/views/stock_picking.xml]: We can now use <odoo>

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by tafaRU** [comment]
thanks @JayVora-SerpentCS, I addressed your comments and rebased. Please check it.

**Review by JayVora-SerpentCS — APPROVED** [review]

**Review by mrcast — APPROVED** [review]
Functional test ok

**Review by astirpe — APPROVED** [review]
Code review only

**Review by hurrinico — APPROVED** [review]
👍 LGTM

**Comment by JayVora-SerpentCS** [comment]
Good to close now @rafaelbn

**Pull request merged**